src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+/*Reset default margins*/
+html,
+body,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --link-active-color: #00acdc;
+  --link-color: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  height: auto;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  background-color: #fff;
+  margin-right: -20px;
```

Comment:
`-20`? what?? maybe just do not set margin-right for last child in that list?

Reply:
✅ Done. Put margin-right to all children except last one.